src/gax-internal/tests/http_retry_loop.rs

```diff
@@ -127,6 +178,29 @@ mod tests {
             .execute::<serde_json::Value, serde_json::Value>(builder, Some(body), options)
             .await;
         assert!(response.is_err(), "{response:?}");
+
+        let spans = TestLayer::capture(&guard);
+        if enable_tracing {
+            assert_eq!(spans.len(), 3, "Expected 3 spans");
+            // Span 1 (attempt 0)
+            assert!(
+                !spans[0]
+                    .attributes
+                    .contains_key(semconv::HTTP_REQUEST_RESEND_COUNT)
+            );
+            // Span 2 (attempt 1)
+            assert_eq!(
+                spans[1].attributes.get(semconv::HTTP_REQUEST_RESEND_COUNT),
+                Some(&"1".to_string())
```

Comment:
I find something like:


```suggestion
                spans[1].attributes.get(semconv::HTTP_REQUEST_RESEND_COUNT).as_deref(),
                Some("1")
```

more readable.